### DISCUSSION TITLE
Problem running LightTable - Harfuzz version too old

### DISCUSSION THREAD

**Discussion by ihojmanb** [LightTable/LightTable#2511] [Q&A] [answered]
I'm trying to run LightTable on ubuntu 20, but I'm having this error:
`(lighttable:76939): Pango-ERROR **: 02:27:58.325: Harfbuzz version too old (1.0.0)`

anyone with the same problem? seems like an Electron issue

first time using LightTable and I really want to give it a try!!! but its getting tedious!!!

**Comment by prertik** [comment, marked as answer]
Yes, this is a problem with using the old version of the electron. There's work ongoing to update electron and all other toolings and a new release: https://github.com/LightTable/LightTable/discussions/2506#discussion-51012

**Reply by ihojmanb** [reply]
Thanks for answering. Didn't knew you were a solo programmer on LT! Seems I Will have to wait till summer 2021 to give it a try :(.
